Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -1966,6 +1984,15 @@ void SoftwareGLContext::gl_pixel_store(GLenum pname, GLfloat param)
     }
 }
 
+void SoftwareGLContext::gl_scissor(GLint x, GLint y, GLsizei width, GLsizei height)
+{
+    RETURN_WITH_ERROR_IF(width < 0 || height < 0, GL_INVALID_VALUE);
+
```

Comment:
```suggestion
APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_scissor, x, y, width, height);
```

Reply:
This needs to be added to the current display list.

Reply:
Added!